apps/poker-roll/src/app/historyPage.jsx

```diff
@@ -1,32 +1,53 @@
-import React from "react";
+import React from 'react';
 import Box from '@mui/material/Box';
 import Container from '@mui/material/Container';
-import { useGetAllSessions } from "./queries/helpers";
+import { useGetAllSessions } from './queries/helpers';
+
+const dayjs = require('dayjs');
 
 // history page will show all past sessions
 
 const PastSession = () => {
-    const sessionsQuery = useGetAllSessions()
-
-    return sessionsQuery.data?.map(session => {
-            return (
-                <Container maxWidth="sm">
-                    <Box>
-                        <h4>{session.startTime}</h4>
-                        <p>{session.game}</p>
-                        <p>{session.casino}, {session.location}</p>
-                        <span>{session.plusMinus}</span>
-                    </Box>
-                </Container>
-    )
-})};
+  const sessionsQuery = useGetAllSessions();
 
-const History = (props) => {
+  return sessionsQuery.data?.map((session) => {
+    if (!session.endTime) {
+      return null;
+    }
+    const start = dayjs(session.startTime);
+    const end = dayjs(session.endTime);
+    const duration = end.diff(`${start}`, 'hour', true).toFixed('1');
```

Comment:
Shouldn't need to put `start` in a template literal here, I think a lot of the dayjs examples use date strings but they can handle date objects as well.

Reply:
removed, still works the same as before